vega_sim/scenario/ideal_market_maker_v2/agents.py

```diff
@@ -122,21 +128,20 @@ def initialise(self, vega: VegaServiceNull):
         )
         self.vega.wait_fn(5)
 
-        ccy_name = f"tDAI{self.tag}"
         # Create asset
         self.vega.create_asset(
             self.wallet_name,
-            name=ccy_name,
-            symbol=ccy_name,
+            name=self.asset_name,
+            symbol=self.asset_name,
             decimals=self.adp,
-            max_faucet_amount=5e10,
+            max_faucet_amount=1e20,
         )
         self.vega.wait_fn(5)
         self.vega.wait_for_datanode_sync()
         # Get asset id
-        self.tdai_id = self.vega.find_asset_id(symbol=ccy_name)
+        self.tdai_id = self.vega.find_asset_id(symbol=self.asset_name)
         # Top up asset
-        self.initial = 100000
+        self.initial = 1e15
```

Comment:
Similarly for this can we promote it to being passed in to the agent as a parameter?